driver.ts

```diff
@@ -36,13 +36,33 @@ export async function reverse(
   await sendText(device, "\x1DB" + enabled);
 }
 
+/**
+ * Select an endpoint from the currently selected device configuration by name
+ * 
+ * @param name usually 'out' or 'in'
+ * @param device the connected USBDevice
+ */
+function selectEndpoint(name: string, device: USBDevice) {
```

Comment:
can you update this type to 'in'|'out'

it will look like

```ts
(name: "in"|"out")
```

Reply:
had a look at the spec. yes! also it's not a name but a direction.